xml/System.Formats.Cbor/CborWriter.xml

```diff
@@ -729,9 +903,27 @@
         <Parameter Name="definiteLength" Type="System.Nullable&lt;System.Int32&gt;" />
       </Parameters>
       <Docs>
-        <param name="definiteLength">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="definiteLength">Writes a definite-length array if inhabited, or an indefinite-length array if <see langword="null" />.</param>
```

Comment:
Could this be reworded to "The length of the definite-length array, or null for an indefinite-length array"?

Reply:
That would make more sense. @eiriktsarpalis would you agree?

Reply:
I think it makes sense, but we might need to update the summary section as well:
"Writes the start of a definite or indefinite-length array (major type 4)."